examples/contrastive-image-text/run_clip.py

```diff
@@ -260,6 +388,7 @@ def main():
         revision=model_args.model_revision,
         use_auth_token=True if model_args.use_auth_token else None,
     )
+    gaudi_config.use_fused_adam = False
```

Comment:
Out of curiosity, why disabling fused ADAM? Is it not compatible with the mediapipe?

Reply:
With mediapipe dataloader, using fused adam optimizer will OOM in case:
- Gaudi 2
- Batch size = 512

Reply:
I add a condition of using mediapipe to disable fused adam